mobile/lib/main.dart

```diff
@@ -59,10 +60,11 @@ class MainPage extends StatelessWidget {
       color: context.colors.backgroundLight,
       width: double.infinity,
       height: double.infinity,
-      child: SingleChildScrollView(
+      child: const SingleChildScrollView(
         child: Padding(
-          padding: const EdgeInsets.only(
+          padding: EdgeInsets.only(
             bottom: RaPageConstraints.radioPlayerHeight,
+            top: 8,
```

Comment:
add constant for this in `lib/resources/ra_page_constraints.dart`

Reply:
Found another suitable solution actually